packages/client/src/components/FullscreenProvider/index.tsx

```diff
@@ -0,0 +1,157 @@
+  'mozfullscreenerror',
+]
+
+const ms = [
+  'msFullscreenEnabled',
+  'msFullscreenElement',
+  'msRequestFullscreen',
+  'msExitFullscreen',
+  'MSFullscreenChange',
+  'MSFullscreenError',
+]
+
+const document: any = // eslint-disable-line
+  typeof window !== 'undefined' && typeof window.document !== 'undefined'
+    ? window.document
+    : {} // eslint-disable-line
+
+const vendor =
+  ('fullscreenEnabled' in document && Object.keys(key)) ||
+  (webkit[0] in document && webkit) ||
+  (moz[0] in document && moz) ||
+  (ms[0] in document && ms) ||
+  []
+
+class fscreen {
+  static requestFullscreen(element: any) {
+    // eslint-disable-line
+    return element[vendor[key.requestFullscreen]]()
+  } // eslint-disable-line
+  static requestFullscreenFunction(element: any) {
+    // eslint-disable-line
+    return element[vendor[key.requestFullscreen]]
+  } // eslint-disable-line
+  static get exitFullscreen() {
+    return document[vendor[key.exitFullscreen]].bind(document)
+  }
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
+  static addEventListener(type: any, handler: any, options: any) {
+    // eslint-disable-line
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
+    return document.addEventListener(vendor[key[type]], handler, options)
+  }
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
+  static removeEventListener(type: any, handler: any, options: any) {
+    // eslint-disable-line
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
+    return document.removeEventListener(vendor[key[type]], handler, options)
+  }
+  static get fullscreenEnabled() {
+    return Boolean(document[vendor[key.fullscreenEnabled]])
+  }
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
+  static set fullscreenEnabled(_) {}
+  static get fullscreenElement() {
+    return document[vendor[key.fullscreenElement]]
+  }
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
+  static set fullscreenElement(_) {}
+  static get onfullscreenchange() {
+    return document[`on${vendor[key.fullscreenchange]}`.toLowerCase()]
+  }
+  static set onfullscreenchange(handler) {
+    document[`on${vendor[key.fullscreenchange]}`.toLowerCase()] = handler
+  }
+  static get onfullscreenerror() {
+    return document[`on${vendor[key.fullscreenerror]}`.toLowerCase()]
+  }
+  static set onfullscreenerror(handler) {
+    document[`on${vendor[key.fullscreenerror]}`.toLowerCase()] = handler
+  }
+}
+
+type Props = {
+  children: ReactNode
+  onChange?: (state: boolean) => void
+  onClose?: () => void
+  onOpen?: () => void
+  enabled?: boolean
+}
+
+export const FullscreenProvider = ({
+  children,
+  onChange,
+  onOpen,
+  onClose,
+  enabled,
+}: Props) => {
+  let fullscreenRef: HTMLElement | null = null
+
+  const handleChange = () => {
+    if (onChange) {
+      onChange(!!fscreen.fullscreenElement)
+    }
```

Comment:
[Предложение]
Здесь можно избавиться от условия и поставить `?.` к функции
```suggestion
    onChange?.(!!fscreen.fullscreenElement)
```